### ISSUE TITLE
tensorflowWithoutCuda

### ISSUE THREAD

**Issue by hacker1024** [SomeoneSerge/nixpkgs-unfree#16]
The `tensorflowWithoutCuda` derivation changes slightly when `config.cudaSupport` is enabled, as some of its dependencies use CUDA.

Can this variant be added as a CI job?

**Comment by SomeoneSerge** [comment]
Sure, can you please submit the patch? Can you also open a nixpkgs issue about the attribute not being stable under `config.cudaSupport`? It shouldn't change, ideally

**Comment by hacker1024** [comment]
I believe it should change, as TensorFlow has dependencies that use CUDA. If it were to override those, it might lead to conflicts with other packages in a larger CUDA-enabled Python environment.

**Comment by SomeoneSerge** [comment]
Yes, it's likely that `tensorflowWithoutCuda` would end up being non-composable with many other python packages. It might still work for python "applications". It'll also become less of a concern once adisbladis's work (requiredPythonModules) gets merged. We might want to reconsider whether `tensorflowWithoutCuda` is making a promise it can't actually keep.

All of those are nixpkgs concerns
